examples/tutorials/analysis-time/Variability_estimation.py

```diff
@@ -0,0 +1,266 @@
+# paper](https://ui.adsabs.harvard.edu/abs/2016A&A...588A.103B/abstract)) as it just computes the level of tension between the lowest and highest measured fluxes in the lighcurve
+
+flux = lc_1d.flux.data
+flux_err = lc_1d.flux_err.data
+
+f_mean = np.mean(flux)
+f_mean_err = np.mean(flux_err)
+
+f_max = flux.max()
+f_max_err = flux_err[np.argmax(flux)]
+f_min = flux.min()
+f_min_err = flux_err[np.argmin(flux)]
+
+ampl_max = (f_max - f_max_err) - (f_min - f_min_err)
+
+ampl_sig = ampl_max / np.sqrt(f_max_err**2 + f_min_err**2)
+
+print(ampl_sig)
+
+# There are other methods based on the peak-to-trough difference to asses the varability in a lighcurve. Here we present as example the relative variability amplitude ([reference paper](https://iopscience.iop.org/article/10.1086/497430)):
+
+RV_a = (f_max - f_min) / (f_max + f_min)
+
+RV_a_err = (
+    2
+    * np.sqrt((f_max * f_min_err) ** 2 + (f_min * f_max_err) ** 2)
+    / (f_max + f_min) ** 2
+)
+
+sig_RV_a = RV_a / RV_a_err
+
+print(sig_RV_a)
+
+# And the variability amplitude ([reference paper](https://ui.adsabs.harvard.edu/abs/1996A%26A...305...42H/abstract)):
+
+A_mp = np.sqrt((f_max - f_min) ** 2 - 2 * f_mean_err**2)
+
+A_mp_100 = 100 * A_mp / f_mean
+
+A_mp_err = (
+    100
+    * ((f_max - f_min) / (f_mean * A_mp_100 / 100))
+    * np.sqrt(
+        (f_max_err / f_mean) ** 2
+        + (f_min_err / f_mean) ** 2
+        + ((np.std(flux, ddof=1) / np.sqrt(len(flux))) / (f_max - f_mean)) ** 2
+        * (A_mp_100 / 100) ** 4
+    )
+)
+
+sig_A_mp = A_mp_100 / A_mp_err
+
+print(sig_A_mp)
+
+######################################################################
+#  Fractional excess variance, point-to-point fractional variance and doubling/halving time
+# ----------------------------------------------
+# The fractional excess variance ([reference
+# paper](https://ui.adsabs.harvard.edu/abs/2003MNRAS.345.1271V/abstract)) is a simple but effective method to assess the significance of a time variability feature in an object, for example an AGN flare. It is important to note that it requires gaussian errors to be applicable. The excess variance computation is implemented in the `estimators.utils` subpackage of `gammapy`. A similar estimator is the point-to-point fractional variance, which samples the lightcurve with a smaller time granularity. In general, the point-to-point fractional variance being higher than the fractional excess variance is a sign of the presence of very short timescale variability. The point-to-point variability is also implemented in the `estimators.utils` subpackage of `gammapy`.
+#
+# In the same subpackage `gammapy` also offers the computation of the doubling and halving time of the lightcurve, an estimator which gives information on the shape of the variability feature.
+
+from gammapy.estimators.utils import (
+    compute_lightcurve_doublingtime,
+    compute_lightcurve_fpp,
+    compute_lightcurve_fvar,
+)
+
+fvar_table = compute_lightcurve_fvar(lc_1d)
```

Comment:
A plot of the excess variance with flux is often useful, but does not belong directly in this tutorial I guess